### PR TITLE
Specify default output directory

### PR DESCRIPTION
**Pull request by rootkea**

### CODE REVIEW

File: src/cli.rs [hunk 1]
```diff
@@ -46,6 +46,7 @@ pub fn build_cli() -> App<'static, 'static> {
                         .short("o")
                         .long("output-dir")
                         .takes_value(true)
+                        .default_value("$PROJECT_ROOT/public")
```
Comment (rootkea) [inline src/cli.rs]: Or may be it should be `project_root/public` as `$PROJECT_ROOT` may lead to a confusion of `PROJECT_ROOT` being an acceptable variable by zola?

File: src/cli.rs [hunk 2]
```diff
@@ -46,6 +46,7 @@ pub fn build_cli() -> App<'static, 'static> {
...
                         .help("Outputs the generated site in the given path"),
```
Comment (rootkea) [inline src/cli.rs]: Or may be a change like this:
```
.help("Outputs the generated site in the given path (by default directory named public in project root)"),
```

**Comment by rootkea** [comment]
Also a similar change for `serve -o`

File: src/cli.rs (hunk 2 above)
Comment (Keats) [reply src/cli.rs]: I would prefer that change

Comment (rootkea) [reply src/cli.rs]: I made the suggested change further shortening the string.

File: src/cli.rs [hunk 3]
```diff
@@ -69,7 +69,7 @@ pub fn build_cli() -> App<'static, 'static> {
                         .short("o")
                         .long("output-dir")
                         .takes_value(true)
-                        .help("Outputs the generated site in the given path"),
+                        .help("Outputs the generated site in the given path (by default 'public' dir in project root)"),
```
Comment (Keats) [inline src/cli.rs:72]: Thinking about that, `zola serve` doesn't actually outputs the generated site, only the assets. The HTML is kept in memory. I'll tweak the message later.

**Review by Keats — APPROVED** [review]
Thanks!

File: src/cli.rs (hunk 3 above)
Comment (rootkea) [reply src/cli.rs:72]: Just leaving a note here: when you'll tweak the message for `--output-dir` in `zola serve`, also mention the same on https://www.getzola.org/documentation/getting-started/cli-usage/#serve

Currently it says nothing about output_dir only containing the assets and HTML being in memory.

**Pull request merged**